css/opengazettes.scss

```diff
@@ -0,0 +1,3 @@
+.jumbotron h1 {
+  color: fuchsia !important;
```

Comment:
If the sass uses $variables and we want to change those, then we need to come first. If we want to override predefined styles, we want to come last. Can we get both? If not, then I think being included last is more flexible.